packages/chat/components/chat/src/chat.template.ts

```diff
@@ -0,0 +1,133 @@
+
+/**
+ * Lit template for Chat interface
+ *
+ * @param {object} customElementClass Class functionality for the custom element
+ * @returns {TemplateResult<1>} Lit html template
+ */
+export function chatTemplate(customElementClass) {
+  const {
+    _handleInput: handleInput,
+    _sendInput: sendInput,
+    _setMessageText: setMessageText,
+    _messages: messages,
+    _messageText: messageText,
+    _queryInProgress: queryInProgress,
+    _handleHoverIn: handleHoverIn,
+    _handleHoverOut: handleHoverOut,
+    _handleEdit: handleEdit,
+    _handleFeedback: handleFeedback,
+    _handleAPIselection: handleAPIselection,
+  } = customElementClass;
+
+  return html`<div class="${c4aiPrefix}--chat-container">
+      <div class="chat-messages">
+        &nbsp;
+        ${messages.map(
+          (message, index) => html` <div
+            class="${message.origin}-message subtype-${message.style}"
+            @mouseout="${(e) => handleHoverOut(index, e, message.origin)}">
+            <div
+              class="message-container"
+              @mouseover="${(e) => handleHoverIn(index, e, message.origin)}">
+              <div class="message-time">${message.time}</div>
+              <div class="message-icon">
+                <div class="${message.origin}-icon">
+                  ${message.origin == 'user' ? User24() : 'AI'}
+                </div>
+              </div>
+              <div class="message-text">${message.text}</div>
+            </div>
+            <div class="${message.origin}-dropdown">
+              <div
+                class="small-button"
+                @click="${(e) => {
+                  handleEdit(e, index);
+                }}">
+                ${Edit16()}
+              </div>
+              <div
+                class="small-button"
+                @click="${(e) => {
+                  handleFeedback(e, index, 'like', message.text);
+                }}">
+                ${ThumbsUp16()}
+              </div>
+              <div
+                class="small-button"
+                @click="${(e) => {
+                  handleFeedback(e, index, 'dislike', message.text);
+                }}">
+                ${ThumbsDown16()}
+              </div>
+            </div>
+          </div>`
+        )}
+        <div class="bot-message ${queryInProgress ? '' : 'hidden'}">
+          <div class="message-container">
+            <div class="loading-container">
+              <div class="loading-bar"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="input-container">
+        <div class="search-icon">${Search24()}</div>
+        <input
+          type="text"
+          .value="${messageText}"
+          @input="${setMessageText}"
+          @keyup="${handleInput}" />
+        <span class="send-button" @click="${() => sendInput(messageText)}">
+          ${ArrowRight16()}
+        </span>
+      </div>
+    </div>
+    <div class="test-section">
```

Comment:
This test section should only in the storybook code... if you want to change it to use controls you can see an example here how to use `select`: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/feat/cwc-storybook-7-vite/packages/carbon-web-components/src/components/tag/tag.stories.ts#L61 and storybook docs: https://storybook.js.org/docs/essentials/controls